lib/ecto_ordered.ex

```diff
@@ -156,13 +156,14 @@ defmodule EctoOrdered do
     end
   end
 
-  defp nearby_query(options, cs) do
+  defp nearby_query(options, cs, direction \\ :asc) do
     options
     |> rank_query
     |> scope_query(options, cs)
     |> select_rank(options.rank_field)
     |> limit(2)
-    |> order_by(^options.rank_field)
+    |> Ecto.Query.exclude(:order_by)
```

Comment:
This is needed because rank_query already sets the order to be ascending on the rank field